recipes/visions/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "visions" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ff3f8daef8417826e39a4ddbd0fb889990b7d5e98e6e0b37fe863c862af93123
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - attrs
+    - networkx
+    - numpy >=1.14.6
+    - pandas >=0.25.3
+    - pip
+    - python >=3.5
+    - tangled-up-in-unicode >=0.0.3
+  run:
+    - attrs
+    - networkx
+    - numpy >=1.14.6
+    - pandas >=0.25.3
+    - python >=3.5
+    - tangled-up-in-unicode >=0.0.3
+
+test:
+  imports:
+    - visions
+    - visions.application
+    - visions.application.summaries
+    - visions.application.summaries.frame
+    - visions.application.summaries.series
+    - visions.core
+    - visions.core.dtypes
+    - visions.core.implementations
+    - visions.core.implementations.types
+    - visions.core.model
+    - visions.lib
+    - visions.lib.relations
+    - visions.utils
+    - visions.utils.coercion
+    - visions.utils.images
+    - visions.utils.monkeypatches
+  requires:
+    - mypy
+    - pytest >=5.2.0
+    - pytest-black
+    - pytest-mypy
+    - pytest-ordering
+    - pytest-rerunfailures
+    - pytest-runner
+    - pytest-sugar
+    - pytest-tldr
+    - imagehash
```

Comment:
It doesn't seem as though you are actually running pytest. I am not sure you need these